src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const CENTURY_EDGE = 30;
+  const XXI_CENTURY = 20;
+  const XX_CENTURY = 19;
+  const SHORT_YEAR_FORM = 'YY';
+  const LONG_YEAR_FORM = 'YYYY';
+
+  const dateParts = date.split(fromFormat[3]);
+  const dateObj = {};
```

Comment:
Almoust shure that you could give it better name)

Reply:
I agree, better naming should be chosen here.